.github/workflows/format.yaml

```
@@ -21,11 +21,34 @@ on:
   pull_request:
 jobs:
   dump-contexts-to-log:
+    permissions:
+      contents: read
+    timeout-minutes: 60
+    permissions:
```

actionlint (1.7.7)

key "permissions" is duplicated in "dump-contexts-to-log" job. previously defined at line:24,col:5

(syntax-check)